CHANGELOG.md

```diff
@@ -8,13 +8,15 @@
     * Added `ObjectGraph` scope to reuse resolved instances 
     * Added `resolveDependencies` method on `DefinitionOf` class to resolve dependencies of resolved instance.  
   [#11](https://github.com/AliSoftware/Dip/pull/11), [@ilyapuchka](https://github.com/ilyapuchka)
-* Added methods to register/remove individual definitions.  
+* Added methods to register/remove individual definitions
   [#11](https://github.com/AliSoftware/Dip/pull/11), [@ilyapuchka](https://github.com/ilyapuchka)
+* All `resolve` methods now can throw error if type can not be resolved [#15](https://github.com/AliSoftware/Dip/issues/15), [@ilyapuchka](https://github.com/ilyapuchka)
```

Comment:
Little nitpicking :wink: The convention we started for CHANGELOG is to end the line describing the change with `.  ` (dot followed by two spaces) then mention the issue/PR number + author on the next line.

```
* Describe the change.␣␣
  [#n](…), [@author](…)
```